### DISCUSSION TITLE
Transaction api to fetch the status of payment.

### DISCUSSION THREAD

**Discussion by sudhir-pratap-singh** [osiset/laravel-shopify#1230] [General]
I want to use the order/transaction api to fetch the status of payment whether the payment is success or failed, anyone can help with that? 

![image](https://user-images.githubusercontent.com/111973644/194217033-61bc2b8a-c497-4774-b8d6-6dec4c6e1e35.png)

**Comment by Kyon147** [comment]
You would need to use the order api and the docs show an example 

https://shopify.dev/api/admin-rest/2022-10/resources/order#get-orders-order-id

Then if you wanted a certain status you would just pass it as a param to your api call and make the api call get all orders of that status rather than a specific one.

**Reply by sudhir-pratap-singh** [reply]
What about if the orders are COD then how we can check the payment status as it was a COD order
